web-animations-1/Overview.bs

```diff
@@ -3537,6 +3540,141 @@ Calculating progress {#core-animation-effect-calculations}
   The final output <dfn>iteration progress</dfn> of an [=animation effect=]
   is simply its [=transformed progress=].
 
+Animation Triggers {#triggers}
+-----------------
+
+### Overview ### {#triggers-overview}
+
+An <dfn export>animation trigger</dfn> is used to control the playback
+of its associated [=animation=] for time-driven animations.
+Like animations, [=animation triggers=] are associated with a [=timeline=]
+and are attached to an [=animation attachment range|attachment range=].
+
+Issue: Should there be any effect of triggers on scroll-driven animations?
+
+### Animation Trigger State ### {#trigger-state}
+
+An [=animation trigger=] |trigger| with [=animation trigger type=] |type|
+has an internal <dfn export lt="animation trigger state">state</dfn> that can be one of the following:
+
+<dl dfn-for="animation trigger state" dfn-type=value>
+  <dt><dfn>idle</dfn>
+  <dd>
+    The [=animation effect=] associated with |trigger| remains in
+    its [=animation effect/before phase=] and stays at zero [=animation/start time=].
```

Comment:
I still don't quite get the hierarchy here. This suggests an animation effect is associated with a trigger but previous we said a trigger is associated with an animation. Which is it?

Reply:
Right, the relation here should be indirect via the animation. The trigger is associated to the animation. The required behavior of the animation is to "hold its effect in a 'just before playing' state". I'll try to fix that.

Reply:
Done, I think it's better now.